jina/peapods/grpc/__init__.py

```diff
@@ -1,11 +1,12 @@
 import argparse
 import asyncio
-from typing import Optional, Callable
+from typing import Optional, Callable, Type
 
 import grpc
 from google.protobuf import struct_pb2
 
 from jina.logging.logger import JinaLogger
+
```

Comment:
It is actually weird, in the rest of the project we use relative paths

Reply:
indeed, I checked and it seems we are using both? But absolute should be more correct, no?